Limits character movement in Y direction

Skim74 opened the pull request:
Updated easy move script so bear can’t walk off the ground.

(ps am I doing this pull request thing right?)

Skim74 committed:
Limits character movement in Y direction

Updated easy move script so bear can’t walk off the ground.

fluffywaffles commented:
Usually a PR will reference an Issue, then (like above on this PR) the Issue page will have a little bookmark icon and will say "<username> referenced this issue xx minutes ago" as opposed to doing it the other way around. (Right above this comment it says "Skim74 referenced this pull request xx minutes ago" - that's what I'm saying will show up, but on the issue page.)

That's mostly just because it's less work to reference the issue in the body of the PR. Either way works.

fluffywaffles commented on Assets/Prefabs/EasyMove.cs:
```diff
@@ -17,20 +18,22 @@ public class EasyMove : MonoBehaviour {
 		else
 			movex = 0;
-		if (Input.GetKey (KeyCode.W)) {
+		if (Input.GetKey (KeyCode.W) && moveYDis < 6) {
```
I would like if we could do this without using 'magic numbers'

Is there a way to derive the 'boundary' numbers for `moveYDis` from the GameObject that the character is constrained to?

fluffywaffles committed:
IAttack interface; closes #11

fluffywaffles committed:
Update Attack interface => abstract class

fluffywaffles committed:
Another fix to IAttack to make abstract properties out of fields

Skim74 committed:
Created basic attack script

A basic implementation of the Attack interface, as a Component that can
be placed on a Unit

fluffywaffles committed:
WIP Add BasicAttack to all PlayerUnits

Still need to implement PerformAttack inside of BasicAttack

fluffywaffles committed:
Fix weird spacing glitch in Unit

fluffywaffles committed:
Spacing fix

fluffywaffles committed:
Make perspective of CombatBear a bit better

fluffywaffles committed:
Use more semantic Debug.LogError for error in CombatHealth
